ios-framework/build.gradle.kts

```diff
@@ -75,3 +76,11 @@ tasks.getByName("build").dependsOn(packForXcode)
 
 // Workaround for issues where types defined in iOS native code cannot be referenced in Android Studio
 tasks.getByName("preBuild").dependsOn(tasks.getByName("compileKotlinIos"))
+
+multiplatformSwiftPackage {
+    swiftToolsVersion("5.3")
+    targetPlatforms {
+        iOS { v("14") }
+    }
+    outputDirectory(File(projectDir, "../ios/build/xcframeworks"))
```

Comment:
SPM can't refer outside of Swift Package binary, so place built `XCFramework` to `ios/build/xcframeworks`